src/niivue.js

```diff
@@ -727,8 +677,20 @@ Niivue.prototype.refreshLayers = function(overlayItem, layer, numLayers) {
 		this.gl.texStorage3D(this.gl.TEXTURE_3D, 6, this.gl.R16UI, hdr.dims[1], hdr.dims[2], hdr.dims[3]);
 		this.gl.texSubImage3D(this.gl.TEXTURE_3D, 0, 0, 0, 0, hdr.dims[1], hdr.dims[2], hdr.dims[3], this.gl.RED_INTEGER, this.gl.UNSIGNED_SHORT, imgRaw);
 	}
-	if (overlayItem.global_min === undefined) //only once, first time volume is loaded
-		this.calMinMax(overlayItem, imgRaw)
+	if (overlayItem.global_min === undefined) {//only once, first time volume is loaded
+		if(!wasmLoaded) {
+			wasmPromise.then((WasmModule) => {
+				wasmLoaded = true;
+				Module = WasmModule;
+				Niivue.prototype.robust_range = Module.cwrap('robust_range', 'number', ['number', 'number']);
+			
+				this.calMinMax(overlayItem, imgRaw);
+			});
+		}
+		else {
+			this.calMinMax(overlayItem, imgRaw);
```

Comment:
This is never called (at least from what I can tell in my tests). put a `console.log("ELSE")` and see if it ever shows up in the console.